lib/App/Config/Chronicle.pm

```diff
@@ -393,6 +409,396 @@ sub _add_dynamic_setting_info {
     return;
 }
 
+=head1 SUBROUTINES/METHODS (NEW)
+######################################################
+###### Start new API
+######################################################
```

Comment:
This won' be "new" on the release after this, so no need to indicate as such; this should be indicated in `Changes` instead.

Reply:
Yep, this is fine.  I'd probably highlight the new APIs as a section under `DESCRIPTION`, to support what's described in `Changes`.